src/WebJobs.Script.WebHost/Security/Authorization/NamedAuthLevelAuthorizationHandler.cs

```diff
@@ -10,9 +10,9 @@
 
 namespace Microsoft.Azure.WebJobs.Script.WebHost.Security.Authorization
 {
-    public class NamedAuthLevelAuthorizationHandler : AuthorizationHandler<AuthLevelRequirement, string>
+    public class NamedAuthLevelAuthorizationHandler : AuthorizationHandler<NamedAuthLevelRequirement, string>
```

Comment:
Need to point to separate requirement `NamedAuthLevelRequirement` so only one handler will be executed.